core/services/relay/evm/types/gen/main.go

```diff
@@ -49,7 +49,7 @@ func runTemplate(name, rawTemplate, outputFile string, input any) {
 		panic(err)
 	}
 
-	res, err := format.Source([]byte(br.Bytes()))
+	res, err := format.Source(br.Bytes())
```

Comment:
not sure why I didn't fail lint before, but fixing it in this PR because I just saw it.